libs/sdk-ui-tests-e2e/cypress/integration/03-sdk-ui-charts/pivotTableTransposed.spec.ts

```diff
@@ -0,0 +1,74 @@
+// (C) 2023 GoodData Corporation
+import * as Navigation from "../../tools/navigation";
+import { Table } from "../../tools/table";
+
+describe("Table Component", { tags: ["checklist_integrated_tiger"] }, () => {
+    const table = new Table(".s-table-component-transpose");
+
+    it("should display Metric in row, Column header on top", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-mr-row-left");
+        table.waitLoaded().hasMetricHeaderInRow(1, 1, "Amount").hasMetricHeaderInRow(2, 1, "Amount");
+    });
+
+    it("should display successful with Metric in row", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-mr-row-top");
+        table.waitLoaded().hasMetricHeaderInRow(1, 1, "Amount").hasMetricHeaderInRow(2, 1, "Amount");
+    });
+
+    it("should show Row attribute and Column header on top", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-rc-row-left");
+        table.waitLoaded().hasHeader("Product").hasColumnHeaderOnTop("Forecast Category");
+    });
+
+    it("should show Row header on top", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-r-row-left");
+        table.waitLoaded().hasHeader("Product");
+    });
+
+    it("should show Metric in row", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-m-row-left");
+        table.waitLoaded().hasMetricHeaderInRow(0, 0, "Amount");
+    });
+
+    it("should show Column header on top", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-c-left");
+        table.waitLoaded().hasColumnHeaderOnTop("Forecast Category");
+    });
+});
+
+describe("Insight View", { tags: ["checklist_integrated_tiger"] }, () => {
+    const table = new Table(".s-insight-view-transpose");
+
+    it("should display Metric in row, Column header on left", () => {
+        Navigation.visit("insight/insight-transpose-has-mc-row-left");
+        table.waitLoaded().hasCellValue(0, 0, "Forecast Category").hasMetricHeaderInRow(1, 0, "Amount");
+    });
+
+    it("should display Metric in column, Column header on top", () => {
+        Navigation.visit("insight/insight-transpose-has-mc-column-top");
```

Comment:
The insightView "insight-transpose-has-mc-column-top", but the config is "column - lefts", so please rename this to column - left